### ISSUE TITLE
[9457] Add way to merge tickets

### ISSUE THREAD

**Issue by Andrew-Lahikainen** [DeskDirector/UserVoice#48]

**Comment by dr3dgen** [comment]
Yes please - this feature becomes essential when there are duplicate tickets sent by multiple users or when a user wants to update a ticket, but creates a new one instead. Keeping track of all of the tickets, details, and who is responsible can get messy without this feature. Please consider.

**Comment by FluidNets-DamianS** [comment]
Also keep in mind that CW does this two different ways (Merge vs Bundle).

**Comment by Andrew-Lahikainen** [comment]
Hey @FluidNets-DamianS out of interest - could you please explain the differences between these two and when you might use one over the other?

I was hoping if we did this feature that we wouldn't have to use ConnectWise specific behaviour. For example, we could simply close the ticket and add a link to the other ("parent") ticket. We could also add the contacts who are on the child ticket as CCs to the parent ticket so they can still keep up to date.

This is kind of like what Zendesk does, and it seems much more simple and seems to avoid some problems. However, in Zendesk a closed ticket cannot be reopened so conversation cannot continue on a merged ticket, it instead creates a "follow-up" ticket.

**Comment by FluidNets-DamianS** [comment]
I don't recall the rules from CW but Bundled tickets keep the child tickets intact but grouped together under a single parent ticket.  Ex:   If 6 different people reported an issue I can bundle them under the first issue but each ticket contact still see's their ticket.  I personally dont like bundled tickets for us most tickets are submitted by a primary contact.  Usually my ticket submitter will report the same issue multiple times:

- Bob Cant Access Email
- Jane cant access the internet
- Nobody can access the server

Three unique issues but are all caused by their file server being offline (that also does DNS and DHCP).  The customer does not know the difference.  If I bundle, it makes it easy for me to update, but the client see's (3) tickets still.   By merging (my preference) the tickets are all consolidated to a single ticket for me and the user.   Only issue is you can un-bundle, there is no un-merge option.

**Comment by Aycorn** [comment]
Similar Request: Service Ticket #21245 - DDTech - Merge & bundle

**Comment by iskolares** [comment]
Similar request #25010 Ability to Merge or Bundle Tickets in DD Tech

**Comment by DevanMorrison** [comment]
We requested the ability to bundle or merge natively in DD per Ticket #25010. Lately especially with the current quarter, duplicate tickets are a bit common given our business sector is going thru the same process at each Company we manage IT for state-wide. 

This feature has also been requested by technicians for quite some time and would be an excellent QoL feature for them to have.

**Comment by iskolares** [comment]
Related Service Ticket #25958

**Comment by iskolares** [comment]
Related ticket #26494

**Comment by MattDunn12** [comment]
The ability to merge/bundle tickets is something that we do on a daily basis, but is something missing on the DD Portal, it is stopping us from using the tech portal instead we're having to use CW for it.

**Comment by iskolares** [comment]
Related request #38762

**Comment by drkmccy** [comment]
Ironically, nine tickets related to this missing feature have been merged into this one. That should be enough to convince the devs of its importance.

**Comment by DevanMorrison** [comment]
Is this a planned feature or anywhere on the roadmap? This has been an asked for feature since we stood up Desk Director in 2019. It seems halfway implemented given Connectwise Parent/Child tickets change the UI & the client knows how they are then related. It just seems odd to have to go to Connectwise to merge items just to go back to Desk Director and work on them. This lack of this one feature is becoming a strain on our technicians -- it's being asked monthly.

**Comment by Nness** [comment]
This is not on the roadmap nor planned.

**Comment by SilverGoLt** [comment]
Quite a important feature, since quite a lot of other Ticket Managment systems have features to allow merging of multiple tickets.

**Comment by Rawns** [comment]
Agree that this should be an addition!

**Comment by DevanMorrison** [comment]
It's interesting DD now has a fan out tool (one ticket creates multiple children tickets), but can't go the other way!

**Comment by Nness** [comment]
To clarify, fan out tool is part of consulting tool chain. It is not part of core product. 

The merge feature mentioned here is about PSA's capability. What I said not on roadmap and nor planned it is also point to use PSA's capability.

Since few years ago, we have stopped expand the dependency on PSA. For example, if we were only used 12 ConnectWise APIs to achieve what we can achieve now, we are not expanding the capability. From recent ConnectWise API defect and how they handle the fix. Until now they have put zero effort to fix it. So if an API is not been maintained why do we want to depends on it?

So it is not against merge functionality but against further depends on unstable API.

We will expand the functionality we can achieve with our own code, database but not depends on others.

**Comment by DevanMorrison** [comment]
Following up here. The connectwise API defect was fixed (2022.2 -- patch version 2022.2.97776). We spoke with our Channel Development Manager during the original issues in February and by mid March this seems to be a non issue.

Given, is this back on the roadmap? If not can it be on the roadmap without the CW APIs (can DD merge and deal with items in the back end). Merging tickets is a core component of any well developed ITSM platform. It's an absolute need for ticket workflows.

**Comment by DevanMorrison** [comment]
We onboarded a new technician two weeks ago & they asked how to merge tickets in DD. This request is over 5 years old now, with several other requests merged into this one (9 in total).

I think I speak for ourselves and other DD users. I would much rather have core capabilities that work than new Teams integrations, etc. This is a basic feature of literally every other competing ticketing system.
